functions/constants.js

```diff
@@ -1,4 +1,5 @@
-module.exports = slugs = [
+exports.testChannel = '771821709589479505';
```

Comment:
We should go ahead and start adding the channel id's that we want to start hitting

Reply:
We'll update that when we release BLACC Smith and as we add more features to it.

Reply:
I think this would be a good time to start the convo about what channels we would like to hit then before just adding them.